### PR TITLE
SUS-4320 Removes INSERT IGNORE option in Block::insert()

### PR DESCRIPTION
**Pull request by leshniak**
https://wikia-inc.atlassian.net/browse/SUS-4320

### CODE REVIEW

File: includes/Block.php [hunk 1]
```diff
@@ -445,8 +445,7 @@ public function insert( $dbw = null ) {
 		$dbw->insert(
 			'ipblocks',
 			$row,
-			__METHOD__,
-			array( 'IGNORE' )
```
Comment (macbre) [inline includes/Block.php]: :+1:

**Review by macbre — APPROVED** [review]

**Review by mszabo-wikia — APPROVED** [review]
Callers expect this method to not throw an exception in case of conflict—let's preserve this guarantee by catching query errors and returning false. The `DBError` constructor will log it for us.

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]
☝️ above was meant to be changes requested

File: includes/Block.php [hunk 2]
```diff
@@ -442,17 +442,22 @@ public function insert( $dbw = null ) {
...
 
-		if ( $affected ) {
-			$auto_ipd_ids = $this->doRetroactiveAutoblock();
-			return array( 'id' => $this->mId, 'autoIds' => $auto_ipd_ids );
+			if ( $affected ) {
+				$auto_ipd_ids = $this->doRetroactiveAutoblock();
+				return array( 'id' => $this->mId, 'autoIds' => $auto_ipd_ids );
+			}
+		}
+		catch ( DBError $ex ) {
```
Comment (mszabo-wikia) [inline includes/Block.php]: let's align the braces otherwise 👍

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**